features/proposal/components/VoteCollapsableHeader/VoteCollapsableHeader.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { FC, useMemo } from 'react';
+import cn from 'classnames';
+import { useTranslation } from 'next-i18next';
+
+import { VoterDetail } from 'features/types';
+import { VotesProgressBar } from 'features/proposal/components/VoicesProgressBar';
+import { Badge, getBadgeVariant } from 'components/Badge';
+import { IconButton } from 'components/button/IconButton';
+
+import styles from './VoteCollapsableHeader.module.scss';
+
+interface VoteCollapsableHeaderProps {
+  setToggle: (newState?: boolean) => void;
+  state: boolean;
+  votes: VoterDetail[];
+  groupName: string;
+}
+
+export const VoteCollapsableHeader: FC<VoteCollapsableHeaderProps> = ({
+  setToggle,
+  state,
+  votes,
+  groupName,
+}) => {
+  const { t } = useTranslation();
+  const voiceCounter = useMemo(
+    () => votes.filter(vote => Boolean(vote.vote)).length,
+    [votes]
+  );
+
+  return (
+    <div
+      tabIndex={0}
+      role="button"
+      className={cn(styles.header, { [styles.open]: state })}
+      onClick={() => setToggle(!state)}
+      onKeyPress={() => undefined}
```

Comment:
I consider we should call setToggle here or provide '-1' to tabIndex

Reply:
Ok, I will fix it.